core-model/test/daml.yaml

```diff
@@ -2,16 +2,15 @@
 # SPDX-License-Identifier: Apache-2.0
 
 sdk-version: 1.17.1
-name: account-hierarchy-trigger
+name: account-hierarchy-core-model-test
 source: src
-version: 0.1.0
+version: 0.1.1
 dependencies:
   - daml-prim
   - daml-stdlib
   - daml-script
-  - daml-trigger
-  - ../lib-finance/model/.daml/dist/finlib-2.0.0.dar
-  - ../core/.daml/dist/account-hierarchy-core-0.1.0.dar
+  - ../../lib-finance/model/.daml/dist/finlib-2.0.0.dar
```

Comment:
Is it required to have finlib in the dependency as AH core already refers to finlib?

Reply:
Yes, I discovered the Daml compiler forces you to name all dependencies in daml.yaml (even the in-direct/transient dependencies like this).